### PR TITLE
Docs: Perlmutter Early Science

### PR DESCRIPTION
**Pull request by ax3l**
Document how to perform large runs.

### CODE REVIEW

File: Tools/BatchScripts/batch_perlmutter.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@
 #SBATCH -N 4
 #SBATCH -J WarpX
 #SBATCH -A <proj>
+# for m3906 LBNL/AMP users: for large runs, comment in
+#S BATCH -q early_science
```
Comment (EZoni) [inline Tools/BatchScripts/batch_perlmutter.sh]: Is this a typo?
```suggestion
#SBATCH -q early_science
```

Comment (ax3l) [reply Tools/BatchScripts/batch_perlmutter.sh]: No, intentionally spaced is it is commented out (see line above).

Not everyone has access to `-q early_science`

Comment (kngott) [reply Tools/BatchScripts/batch_perlmutter.sh]: To me, that reads as a possible typo instead of a comment. I'd do double (or more) hashes: `##SBATCH`.

Comment (ax3l) [reply Tools/BatchScripts/batch_perlmutter.sh]: Cool, yes that works if double-hashes are commented out.
Cool!
```suggestion
##SBATCH -q early_science
```

Ref.: https://hpc.nmsu.edu/discovery/slurm/slurm-commands/

**Review by EZoni — APPROVED** [review]

**Pull request merged**